docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -167,6 +167,11 @@ Use Playwright to simulate and validate user workflows including:
 
 Visit the https://playwright.dev/docs[Playwright documentation] for information.
 
+[NOTE]
```

Comment:
This is a bit confusing for me, I've used headful mode for debugging on synthetics cli, shouldn't this specifically refer to Elastic hosted synthetics service/heartbeat monitors instead?

Running headful locally used to be supported. IMO, it should still be

Reply:
@emilioalvap that's correct, it is still possible via the CLI, which is why I didn't add the note [there](https://www.elastic.co/guide/en/observability/current/synthetics-command-reference.html#elastic-synthetics-command)

I also specifically mentioned `...run browser monitors...` to try and distinguish this as the monitor (not running the journey locally for debugging purposes), but it sounds like this was still confusing.  I didn't want to add a link here to the CLI saying it was still possible, as I thought it would get too confusing, but do you think that would help then?

Reply:
I think the wording on the CLI specific page is quite detailed, I'd replicate that here:
```
Do not attempt to run in headful mode when running through Elastic’s global managed testing infrastructure or Private Locations as this is not supported.
```

IMO, this page is more generic than the CLI one to simply state that headful mode is not supported, it might be confusing for users as well.